www/docs/constructs/AppSyncApi.about.md

```diff
@@ -734,3 +734,52 @@ new AppSyncApi(stack, "GraphqlApi", {
   },
 });
 ```
+
+### Batching GraphQL Resolvers (N+1)
+
+Using naive GraphQL resolvers may result in what the GraphQL community calls the "N+1 Problem", which occurs when the system makes too many upstream requests for a given query.
+
+For example, imagine that an an incoming request is asking for 50 `Company` objects, each with its associated list of `Office` objects. Naive GraphQL resolvers will make a request to fetch the 50 `Company` records first, then repeatedly request the `Office` objects for the each of the 50 `Company` records. This results in 51 requests in total.
```

Comment:
There seem to be 2 "an" before "incoming"

Reply:
@adikari Thanks for the feedback - fixed.